go.mod

```
@@ -9,8 +9,8 @@ require (
 	github.com/hako/durafmt v0.0.0-20191009132224-3f39dc1ed9f4
 	github.com/hashicorp/raft v1.1.1
 	github.com/hashicorp/raft-boltdb v0.0.0-20191021154308-4207f1bf0617
-	github.com/liftbridge-io/go-liftbridge v0.0.0-20191106180712-4f7e4b9d8611
-	github.com/liftbridge-io/liftbridge-api v0.0.0-20190910222614-5694b15f251d
+	github.com/liftbridge-io/go-liftbridge v0.0.1-alpha.0.20200120163431-bfb99c041e6f
```

You'll have to update this:
```suggestion
	github.com/liftbridge-io/go-liftbridge v0.0.1-alpha.0.20200215001001-1f9104ec55d9
```